lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_get_latest_release.rb

```diff
@@ -0,0 +1,110 @@
+require 'fastlane/action'
+require_relative '../client/firebase_app_distribution_api_client'
+require_relative '../helper/firebase_app_distribution_auth_client'
+require_relative '../helper/firebase_app_distribution_helper'
+
+module Fastlane
+  module Actions
+    class FirebaseAppDistributionGetLatestReleaseAction < Action
+      extend Auth::FirebaseAppDistributionAuthClient
+      extend Helper::FirebaseAppDistributionHelper
+
+      def self.run(params)
+        auth_token = fetch_auth_token(params[:service_credentials_file], params[:firebase_cli_token])
+        fad_api_client = Client::FirebaseAppDistributionApiClient.new(auth_token, params[:debug])
+
+        UI.message("⏳ Fetching latest release for app #{params[:app]}...")
+
+        releases = fad_api_client.list_releases(app_name_from_app_id(params[:app]), 1)[:releases] || []
+        if releases.length == 0
+         	UI.user_error!("No releases for app #{params[:app]} found in App Distribution.")
```

Comment:
Hmm I wonder if we shouldn't error here. A possible valid use case is to try and get the latest release only to find out there isn't one, and then they could just move forward and upload the binary.

Reply:
So we could return `nil` in that case? That sounds reasonable to me.

Reply:
SGTM!